lib/jekyll/tags/highlight.rb

```diff
@@ -83,6 +85,7 @@ def parse_options(input)
         end
 
         options[:linenos] = "inline" if options[:linenos] == true
+        options[:start_line] = Integer(options[:start_line]) unless options[:start_line].nil?
```

Comment:
nit: I'd personally try to avoid `unless` statements because they're a bit harder to wrap your head around than (in my opinion) an `if !(condition)`.

Reply:
@oe I _believe_ that Rubocop complains about `if ! condition` and suggests using `unless condition` instead.

Reply:
Indeed, @pathawks got it right:

<img width="720" alt="3__bash_and_1__bash" src="https://user-images.githubusercontent.com/49583/41671570-81184330-7485-11e8-9bc7-759fadc944b6.png">

I don’t feel strongly if there is another preferred way to express this.

Reply:
@pathawks dangit!

Reply:
disregard my nit then